rasa/core/training/generator.py

```diff
@@ -179,7 +179,7 @@ def __init__(
             max_number_of_augmented_trackers=max_number_of_augmented_trackers,
             tracker_limit=tracker_limit,
             use_story_concatenation=use_story_concatenation,
-            rand=random.Random(42),
+            rand=42,  # send random seed
```

Comment:
what's wrong with `random.Random(42)`?

Reply:
Nothing. I must have mistaken it for the cause of randomisation while something else was. My bad. Changing it back to the was it was.